pkg/model/actionhttp.go

```diff
@@ -25,13 +25,14 @@ import (
 // ActionHTTP represents a HTTP request with retry logic.
 type ActionHTTP struct {
 	Action
-	URL       URL               `json:"url"`
-	Method    string            `json:"method,omitempty"`
-	Attempts  uint              `json:"attempts,omitempty"`
-	Delay     string            `json:"delay,omitempty"`
-	DelayType string            `json:"delay_type,omitempty"`
-	Header    map[string]string `json:"header,omitempty"`
-	Body      interface{}       `json:"body,omitempty"`
+	URL            URL               `json:"url"`
+	CloudAuthToken bool              `json:"cloud_auth_token"`
```

Comment:
should it placed here? cuz this `ActionHTTP` is used for sending request in general, not cloud/openstack specific.

Reply:
Yes, it's optional. By default, it is disabled. It allows user to use Cloud authentication service for Action URL.

Do you have any suggestions?